lib/views/help/privacy.html.erb

```diff
@@ -1,39 +1,43 @@
 <div id="left_column_flip">
   <div class="page__header">
     <h1 class="page__header__title"><%= @title %></h1>
   </div>
-  <p id="public_request">On AsktheEU.org, only your name is visible to the general public unless you put other personal information in the body of the request.</p>
-  <p>We strongly advise you not to put any personal information in the body of the request. That includes not putting your phone number, home address, email address or other personal or family or work details.</p>
-  <p>The AsktheEU.org website is run by Access Info Europe and we take your privacy very seriously. We comply with the <a href="https://gdpr-info.eu/">European Union's General Data Protection Regulation</a>.</p>
-  <h2>What personal data of mine will you process?</h2>
-  <p>Access Info receives your full name and your email address. We will use these for managing the AsktheEU.org website. We never share your email address with anyone else without your express permission.</p>
-  <p>The AsktheEU.org website transfers your name in access to documents requests to European Union bodies. By using the AsktheEU.org website, you expressly give your permission for us to transfer your name to the relevant EU body. You also expressly give your permission for that EU body to process your name for the purpose of answering your request. That means that the name will be shared with relevant European Union officials who are involved in searching for the requested documents and in preparing and signing the response to you.</p>
-  <p>We make clear to the EU that in no circumstances have you given permission to the EU to transfer your name to any other third party without your express consent.</p>
+  <p id="public_request">On AsktheEU.org, as a general rule only your name is visible to the general public and we advise you not to put any personal information in the body of the request.</p>
+  <p>The AsktheEU.org website is run by Access Info Europe and we take privacy and personal data protection seriously. We comply with the <a href="https://commission.europa.eu/law/law-topic/data-protection/data-protection-eu_en">European Union's General Data Protection Regulation</a>.</p>
+<h4>Exceptions to the no personal data rule</h4>
+<p>The <strong>European Commission</strong> does ask for your postal address. You can add it at the time that you make the request or when they come back to you and ask you to provide it. As soon as you have sent the mail with the postal address, contact the AsktheEU.org team on team at asktheeu.org and we will redact that information.</p>
+<p>The only other body which currently asks for personal information is <strong>Frontex</strong>, the <a href="https://www.frontex.europa.eu/media-centre/news/news-release/frontex-to-provide-additional-support-to-bolster-finland-s-borders-41el4Z">European Border and Coast Guard Agency</a>, which asks that you send them via a separate email a copy of your identity document. This will not become public. </p>
```

Comment:
```suggestion
  <h4>Exceptions to the no personal data rule</h4>
  <p>The <strong>European Commission</strong> does ask for your postal address. You can add it at the time that you make the request or when they come back to you and ask you to provide it. As soon as you have sent the mail with the postal address, contact the AsktheEU.org team on team at asktheeu.org and we will redact that information.</p>
  <p>The only other body which currently asks for personal information is <strong>Frontex</strong>, the <a href="https://www.frontex.europa.eu/media-centre/news/news-release/frontex-to-provide-additional-support-to-bolster-finland-s-borders-41el4Z">European Border and Coast Guard Agency</a>, which asks that you send them via a separate email a copy of your identity document. This will not become public.</p>
```